### DISCUSSION TITLE
部分机型出现花屏现象

### DISCUSSION THREAD

**Discussion by koxyoxyo** [Tencent/libpag#1988] [Q&A]
花屏PAG和表现视频
[花屏pag.zip](https://github.com/Tencent/libpag/files/13645317/pag.zip)

目前已知有问题机型：FNE-AN00 荣耀70  android 12版本 (MagicOS 7.0)
PAG版本：4.2.110

代码：
<img width="1011" alt="image" src="https://github.com/Tencent/libpag/assets/49026908/76b09dfe-8981-46e7-8de7-0b21acc3f509">

无报错日志与堆栈，想请教一下问题原因和解决方案

**Comment by CodeJhF** [comment]
抱歉这条当时没有被跟进到，是我们漏了。

因长期无回复，且 libpag 4.3 及更早版本已停止维护（当前仅维护 4.4 / 4.5），先关闭本条。
上报版本：4.2.110。

如果问题在最新版本（4.4 / 4.5）仍能复现，欢迎在本帖回复或新开 issue，并附上 libpag 版本号、平台与系统版本、可复现的 PAG 文件或最小 demo，我们会优先跟进。（stale-close）
